src/controllers/quest.js

```diff
@@ -387,6 +387,10 @@ class Quest extends Controller {
 		if (item.quest_task && !this.config.general.ignoreMADQuestString) {
 			str = item.quest_task
 		} else {
+			if (!Object.keys(this.GameData.translations).includes(language)) {
```

Comment:
I think you could just do `!this.GameData.translations[language]` ?
since that would return undefined?

Reply:
this:
```
			if (!Object.keys(this.GameData.translations).includes(language)) {
				this.log.warn(`Missing language ${language}, fallback to english`)
				language = 'en'
			}
```

to:
`			language = !this.GameData.translations[language] ? 'en' : language`

and no log entry, that it fallback ???

Reply:
a log entry is probably not required so this would work well.
some people don't like ?: as it doesn't scan as well visually; I have no problem with it